commons/src/main/res/values-ko-rKR/strings.xml

```diff
@@ -543,107 +541,104 @@
+    <string name="picasso_title">Picasso (이미지 로딩과 캐싱)</string>
+    <string name="cropper_title">Android Image Cropper (이미지 자르기와 회전)</string>
+    <string name="rtl_viewpager_title">RtlViewPager (Right-to-left 스와이핑)</string>
+    <string name="joda_title">Joda-Time (Java 날짜 대체)</string>
+    <string name="stetho_title">Stetho (데이터베이스 디버깅)</string>
+    <string name="otto_title">Otto (이벤트 bus)</string>
+    <string name="photoview_title">PhotoView (확대 가능한 GIF)</string>
+    <string name="pattern_title">PatternLockView (패턴 보안 기능)</string>
+    <string name="reprint_title">Reprint (지문 보안 기능)</string>
+    <string name="gif_drawable_title">Gif Drawable (GIF 로딩)</string>
+    <string name="autofittextview_title">AutoFitTextView (텍스트 크기 변경)</string>
+    <string name="robolectric_title">Robolectric (프레임워크 테스팅)</string>
+    <string name="espresso_title">Espresso (테스트 도움)</string>
     <string name="gson_title">Gson (JSON parser)</string>
-    <string name="leak_canary_title">Leak Canary (memory leak detector)</string>
-    <string name="number_picker_title">Number Picker (customizable number picker)</string>
-    <string name="exoplayer_title">ExoPlayer (video player)</string>
-    <string name="panorama_view_title">VR Panorama View (displaying panoramas)</string>
-    <string name="sanselan_title">Apache Sanselan (reading image metadata)</string>
-    <string name="filters_title">Android Photo Filters (image filters)</string>
-    <string name="gesture_views_title">Gesture Views (zoomable images)</string>
+    <string name="leak_canary_title">Leak Canary (메모리 누수 감지기)</string>
+    <string name="number_picker_title">Number Picker (편집가능 숫자 선택기)</string>
+    <string name="exoplayer_title">ExoPlayer (영상 재생기)</string>
+    <string name="panorama_view_title">VR Panorama View (파노라마 뷰어)</string>
+    <string name="sanselan_title">Apache Sanselan (이미지 메타데이터 읽기)</string>
+    <string name="filters_title">Android Photo Filters (이미지 필터)</string>
+    <string name="gesture_views_title">Gesture Views (확대 가능한 이미지)</string>
 
     <!-- Google Play listing -->
-    <string name="deprecated_app">DEPRECATED: This version of the app is no longer maintained, get the Pro version at https://play.google.com/store/apps/details?id=com.simplemobiletools.xxx.pro</string>
-    <string name="pro_app_refund">Don\'t forget that if you uninstall any paid app within 2 hours, you will automatically be refunded. If you want a refund anytime later, just contact us at hello@simplemobiletools.com and you will get it. That makes it easy to try it out :)</string>
+    <string name="deprecated_app">구버전 경고: 이 버전의 앱은 더이상 관리되지 않습니다. Pro 버전을 설치해 주세요: https://play.google.com/store/apps/details?id=com.simplemobiletools.xxx.pro</string>
+    <string name="pro_app_refund">플레이스토어에서 구매한 앱은 2시간 내에 삭제하면 자동으로 환불받을 수 있다는 사실을 항상 잊지 마시기 바랍니다. If you want a refund anytime later, just contact us at hello@simplemobiletools.com and you will get it. That makes it easy to try it out :)</string>
```

Comment:
Hey, you havent translated the second half of this string. Anyway, Ill accept it so it is easier to submit the update, just 1 line. Thanks

Reply:
I left that line intentionally because people wouldn't speak English. My expectation was people who only speak Korean would concact you, and later both get puzzled because of the bad internet translator. If you wish, I'll translate it with additional "(영어)" (meaning "English") after the mail address, or without it if you can handle those situations.

Reply:
translate the whole line please